### PR TITLE
Topo2 fix nocodec capture devices

### PR DESCRIPTION
**Pull request by jsarha**
Supposed to fix https://github.com/thesofproject/sof/issues/7093

### CODE REVIEW

**Comment by jsarha** [comment]
The PR has https://github.com/thesofproject/sof/pull/7682 bellow it. I'll rebase as soon as #7682 is merged.

**Comment by jsarha** [comment]
> @jsarha so rebase, as base commit it merged?

Already done.

**Review by juimonen — APPROVED** [review]

File: tools/topology/topology2/cavs-nocodec.conf [hunk 1]
```diff
@@ -83,11 +83,11 @@ Define {
 	SSP1_ENABLED			"true"
 	PASSTHROUGH			"false"
 	SSP0_PCM_NAME			"Port0"
-	SSP0_CAPTURE_PCM		"ssp-capture"
+	SSP0_CAPTURE_PCM		"Port0 2nd Capture"
```
Comment (kv2019i) [inline tools/topology/topology2/cavs-nocodec.conf:86]: Why is this "2nd Capture" , I can't immediately figure out... UPDATE: ok, checked the #7093 .

File: tools/topology/topology2/platform/intel/dmic-generic.conf [hunk 2]
```diff
@@ -1,5 +1,5 @@
 Define {
-       DMIC0_PCM_NAME	"DMIC"
+       DMIC0_PCM_NAME	"DMIC Raw"
```
Comment (kv2019i) [inline tools/topology/topology2/platform/intel/dmic-generic.conf:2]: I guess I'm very late to the came, but given there is gain and EQ, calling this "Raw" is bit of a stretch. Not blocking on this, if the naming has been agreed.

**Review by kv2019i — APPROVED** [review]
@plbossart Good to go?

**Review by ranj063 — APPROVED** [review]

**Comment by kv2019i** [comment]
Unrelated fails in https://sof-ci.01.org/sofpr/PR7694/build8577/devicetest/index.html and https://sof-ci.01.org/sofpr/PR7694/build8576/devicetest/index.html . Proceeding with merge.

**Pull request merged**